logx/logx_test.go

```diff
@@ -57,6 +57,19 @@ func TestDefaultAndLogstashLogging(t *testing.T) {
 	}
 }
 
+func TestLoggingWithCustomSkipLevel(t *testing.T) {
+	assert := assert.New(t)
+	rec := make(recorder, 1)
+	defaultLogger := logx.New(logx.WriterOpt(rec), logx.WithoutTimeOpt(), logx.FileSkipLevel(4))
+
+	log(defaultLogger, "Test")
```

Comment:
why are you creating this ```log()``` func and not just calling directly ```defaultLogger.Debug(message)```

Reply:
This test is to verify that a wrapped log() function will still output the correct line number (line 65 of the test instead of line 70 which is where the defaultLogger.Debug() is called